.gitignore

```diff
@@ -5,6 +5,7 @@ settings-docker.xml
 .sbt/
 .npm/
 .bnd/
+.docker/
```

Comment:
this one is not necessary: you should delete your local directory, current scripts don't store files here

Reply:
Ok, I removed this change.